### PR TITLE
Final schedule

### PR DESCRIPTION
**Pull request by scottyhq**
Generated with `hw_create_schedule -o uwhackweek -r schedule-2024 -t $GITHUB_TOKEN -d ../event-page-2024 -l "All,GeoSMART,SnowEx,ICESat-2"`

### CODE REVIEW

File: schedule.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ tabs:
     - Mark Welden-Smith
   - title: 'Plenary: Best Practices for Collaborative Development'
     description: Effective teamwork and collaborations
-    time: 9:30 - 10:15
+    time: 9:45 - 10:15
```
Comment (scottyhq) [inline schedule.yaml:15]: previous event only scheduled for 9-930 (https://github.com/uwhackweek/schedule-2024/issues/1 - I assume @markweldensmith is presenting. So we actually have a 15 min buffer.

File: schedule.yaml [hunk 2]
```diff
@@ -32,12 +32,6 @@ tabs:
     label: snowex
     leads:
     - H.P. Marshall
-  - title: 'Tutorial: ICESat-2 Mission Overview (Room 345)'
-    description: All about ICESat-2 and the data products
-    time: 11:15 - 12:00
-    label: icesat-2
-    leads:
-    - Tyler Sutterley
```
Comment (jomey) [inline schedule.yaml]: Should this be removed?

Comment (scottyhq) [reply schedule.yaml]: my bad, if it's late and you mistype a label (ICESat-2) those events get deleted

File: schedule.yaml (hunk 1 above)
Comment (naclomi) [reply schedule.yaml:15]: I was discussing the scheduling with @aaarendt last week -- it sounded like having 45 minutes for the opening session would help things feel less rushed, and my session is designed for a 30 minute slot anyway. Ultimately, can go with whatever timing works out naturally tomorrow.

Comment (aaarendt) [reply schedule.yaml:15]: Yes, I updated the welcome session to be 45 min. Charley needs a bit more time for her activity.

**Review by aaarendt — APPROVED** [review]

**Pull request merged**